packages/api-bindings/src/graphql/common.graphql

```diff
@@ -267,6 +252,7 @@ fragment Comment on Comment {
       ...Post
     }
 
+    # we do not want to promote comments on mirrors, should we remove this?
```

Comment:
if we remove the mirror edge on `commentOn` do we need to filter out manually all empty edges when receiving a response in case someone directly called the protocol and puts the comment on the mirror? 

```
__typename: "Comment",
commentOn: {},
...
```

Reply:
I thought about that... I am just not sure when and where to do it reliably. It could be done as `merge` function of all fields that could return publications, but seems daunting and error prone (if we don't add such filtering to a new publications query for example).